.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,100 +1,7 @@
-👋👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.
+👋👋 Hello there! Welcome. Please follow the instruction below. 
 
-1. Select **one of the templates below** that fits your contribution
-2. Remove this text as well as the other templates
-3. Fill in all sections of the template
-4. Summon a reviewr using @mentions (optional)
-5. Click "Create pull request"
+Click the `Preview` tab and select a PR template:
 
----
-
-## Showcase addition
-
-<!--
-
-Provide information for every bullet in the list below. 
-
-Be sure to read the #Showcases section in the CONTRIBUTING.md file. 
-
--->
-
-* Name: *Project name*
-* Description: *Describe your project* (See [Showcase guidelines](CONTRIBUTING/#showcase))
-* Link:   <link_to_project>
-* Source: <link_to_source_code>
-* Tags:
-  * `analytics`
-  * `explorer`
-  * `educational`
-  * `game`
-  * `gateway`
-  * `opensource`
-  * `pooltool`
-  * `metadata`
-  * `nativetoken`
-  * `nft`
-  * `wallet`
-
-<!--
-
-Select at most 3 tags and remove all others. 
-
-The tags you select must match the tags in your changes to the showcase.js
-
---> 
-
----
-
-## Updating documentation
-
-#### Issue
-
-Fixes #issueId
-
-<!--
-
-If there is no issue for your PR, (i.e. typos or new article), remove the *issue* section and just provide the description.
-
--->
-
-#### Description of the change
-
-<!--
-
-We must be able to understand the purpose of your change from this description. If we can't get a good idea of the benefits of the change from the description here, the pull request may be closed at the maintainers' discretion.
-
--->
-
----
-
-## Bugfix
-
-Fixes #issueId
-
-<!-- 
-
-If you found a 0Day bug and there is no issue for it, please [create one now](https://github.com/cardano-foundation/developer-portal/issues/new) and reference it.
-
--->
-
-#### Description
-
-<!--
- 
-Describe what you did to fix the issue
-
---> 
-
----
-
-## Quickfix
-
-<!--
-
-Leave the ##Quickfix header.  
-
-Don't provide any structure to your description or let the title of the PR speak for itself.
-
-Be advised. PRs with no structure have the lowest priority for maintainers to review / merge and might not make it into the next release of the developer portal.
-
--->
+- [Add Showcase](?expand=1&template=showcase.md)
```

Comment:
Did we also want to add auto labels (`&label=showcase`)? Thought that was one of the points why we wanted multiple PR templates

Reply:
yip

Reply:
I can set up auto label bot for the repository in a separate pr that will focus on title and text of the pr. Or would you guys prefer me to do it in this pr as well?

Reply:
either way is good for me 🙂

Reply:
Am I wrong or I'm seeing some change you've suggested on the PR but I can't tell what exactly do you want to change here

Reply:
Ye - if going via bot route, no actions needed on this PR 👍🏻